pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -89,6 +90,35 @@ func (pl *VolumeBinding) Name() string {
 	return Name
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEvent {
+	events := []framework.ClusterEvent{
+		// Pods may fail because the storage class is not configured correctly.
+		// After the storage class is fixed (e.g. allowedTopologies,
+		// volumeBindingMode), the pods may become schedulable.
+		{Resource: framework.StorageClass, ActionType: framework.Add | framework.Update},
```

Comment:
Hold on. Seems we don't get handle StorageClass update event:

https://github.com/kubernetes/kubernetes/blob/1db614ec8fd2cfe7c0ac852dc54468aa094c11ce/pkg/scheduler/eventhandlers.go#L456-L460

Reply:
I guess it's rare to update storage classes in real clusters, but according to API server validation logic `allowedTopologies` and `volumeBindingMode` are mutable, we need the `Update` event here.